hack/run-e2e-test-kind.sh

```diff
@@ -85,11 +107,26 @@ echo "## label KinD's control-plane-node as sriov capable"
 kubectl label node kind-worker feature.node.kubernetes.io/network-sriov.capable=true --overwrite
 echo "## label KinD worker as worker"
 kubectl label node kind-worker node-role.kubernetes.io/worker= --overwrite
-echo "## retrieving netns path from container"
-netns_path="$(docker inspect --format '{{ .NetworkSettings.SandboxKey }}' "${kind_container}")"
-echo "## exporting test device '${test_pf_pci_addr}' and test netns path '${netns_path}'"
-export TEST_PCI_DEVICE="${test_pf_pci_addr}"
-export TEST_NETNS_PATH="${netns_path}"
+if [[ "${INTERFACES_SWITCHER}" == "system-service" ]];then
+    pf="$(ls /sys/bus/pci/devices/${test_pf_pci_addr}/net)"
+    cat <<EOF > /etc/vf-switcher/vf-switcher.yaml
```

Comment:
need to make sure directory exists

Getting the following:

```
## make KinD's sysfs writable (required to create VFs)                                                                                                                                                           ## label KinD's control-plane-node as sriov capable                                                                                                                                                              node/kind-worker labeled                                                                                                                                                                                         ## label KinD worker as worker                                                                                                                                                                                   node/kind-worker labeled                                                                                                                                                                                         ./hack/run-e2e-test-kind.sh: line 112: /etc/vf-switcher/vf-switcher.yaml: No such file or directory 
```

Reply:
Done